docs/en/manuals/compute.md

```diff
@@ -34,7 +34,7 @@ Procedural Generation
 
 Rendering effects
   - Global illumination: Simulating realistic lighting by approximating the way light bounces around a scene.
-  - Voxelisation: Create a 3D voxel grid from mesh data.
+  - Voxelization: Create a 3D voxel grid from mesh data.
```

Comment:
Not sure if we ever agreed on a US vs UK spelling (ie z vs s) and I don't think we are consistent in the docs

Reply:
Yea, a standard is something to think about. It is also inconsistent in the engine and editor console which I've noticed in the past.

```
INFO:DLIB: Initialized Remotery (ws://)
INFO:ENGINE: Initialised sound device 'default'
```